dask/compatibility.py

```diff
@@ -169,6 +169,18 @@ def writelines(self, *args, **kwargs):
     else:
         GzipFile = gzip.GzipFile
 
+try:
+    from lzma import LZMAFile
+except ImportError:
+    try:
+        from backports.lzma import LZMAFile
+    except ImportError:
+        class LZMAFile:
+            def __init__(self, *args, **kwargs):
+                raise ValueError("lzma module not found on system, "
+                                 "consider installing lzma backports")
```

Comment:
Perhaps "LZMA decompression requires the lzma module. To use, install `backports.lzma`."?

Dask also only supports Python 3.3+ for python 3 (which has lzma come standard). So you could move the `from lzma import LZMAFile` line up under the `if PY3` block, and put the other `try:    from backports...` in the python 2 block. Slightly cleaner IMO.